### ISSUE TITLE
[ BUG ] Get-FalconScheduledReport -Execution -Detailed produces 404 error

### ISSUE THREAD

**Issue by bk-cs** [CrowdStrike/psfalcon#143]
**Describe the bug**
When using `Get-FalconScheduledReport` with the `Execution` switch, `Detailed` does not submit the identifier results to the proper API, which leads to an `404: scheduled_report not found` error.

**To Reproduce**
```
Get-FalconScheduledReport -Execution -Detailed
```
**Expected behavior**
Detailed output of an execution result.

**Environment (please complete the following information):**
 - OS: Windows 10
 - PowerShell: 5.1, 7.1.4
 - PSFalcon: 2.1.5

**Comment by bk-cs** [comment]
Until this issue is fixed, you can work around it by performing the requests in two steps:

```
$Executions = Get-FalconScheduledReport -Execution
Get-FalconScheduledReport -Execution -Ids $Executions
```

**Comment by bk-cs** [comment]
Resolved with v2.1.6 release, but `Get-FalconScheduledReport` does require the use of "execution ids" rather than "report ids".
